src/main/java/com/training/assignments/Assignment1.java

```diff
@@ -0,0 +1,32 @@
+package com.training.assignments;
+
+public class Assignment1 {
+    public double intToDouble(int value) {
+        return (double) value;
+    }
+
+    public int doubleToInt(double value) {
+        return (int) value;
+    }
+
+    public float doubleToFloat(double value) {
+        return (float) value;
+    }
+
+    public double floatToDouble(float value) {
+        return (double) value;
+    }
+
+    public long byteToLong(byte value) {
```

Comment:
Please follow the below method naming convention when your method is returning something:
- an number/string -> `get` prefix
- an boolean -> `is` prefix